src/widget/scroll.rs

```diff
@@ -147,11 +235,35 @@ impl<T: Data> Widget<T> for Scroll<T> {
         } else {
             None
         };
+
+        match event {
+            Event::AnimFrame(_) => {
```

Comment:
A couple things here. First, the scroll bars will fade immediately if there's some other widget requesting animation. I'd guard this by the timer id being invalid.

Second, on high refresh rate monitors the fade will be faster. I'd use the  interval to control the fade speed.

Reply:
These are good findings! I wasn't aware of any.

Reply:
Changed.